services/Skyra.Database/Models/Entities/Guild.cs

```diff
@@ -28,6 +28,18 @@ public class Guild
 		[Column("disabled-commands", TypeName = "character varying(32)[]")]
 		public string[] DisabledCommands { get; set; } = Array.Empty<string>();
 
+		[Column("afk.role")]
+		[StringLength(19)]
+		public string? AfkRole { get; set; }
+
+		[Column("afk.prefix")]
+		[StringLength(32)]
+		public string? AfkPrefix { get; set; }
+
+		[Required]
+		[Column("afk.prefix-force")]
+		public bool AfkPrefixForce { get; set; }
```

Comment:
You need to add the default postgres value in the `SkyraDbContext` with `entity.HasDefaultValueSql`